solutions/USACO feb20-triangles.mdx

```diff
@@ -59,4 +59,41 @@ public class triangles{
 ```
 
 </JavaSection>
+
+<CPPSection>
```

Comment:
kinda liked the official for this one better??? feel free to put a modified version of the official sol if u feel like its lacking in some areas

Reply:
Hmm you have a point, we used the same idea... I think I can remove this, how do i do it

Reply:
You can just delete ur impl and commit ig